app/templates/admin/userManagement.html

```diff
@@ -114,36 +121,124 @@ <h3 class="accordion-header" id="headingTwo">
+      <div id="collapseThree" class="accordion-collapse collapse {{ show }}" aria-labelledby="headingThree" data-bs-parent="#adminManagement">
+        <div class="accordion-body ml-5">
+          <div class="d-flex justify-content-center">
+           <table>
+              <tr>
+                 <th class="programHeader">Program</th>
+              </tr>
+              
+              {% for program in programs %}
+              <tr>
+                 <td>
+                  <input class="form-label borderless" 
+                    name="programName" 
+                    type="text" readonly 
+                    value="{{ program.programName }}"></td>
+                 <td>
+                    <button class="btn btn-success test" 
+                    role="button" id="programManagementButton{{ loop.index }}" 
+                    data-bs-toggle="modal" 
+                    data-bs-target="#adminProgramManagement"
+                    onclick="programModal('{{ program.programName }}', '{{ program.programDescription }}', '{{ program.partner }}', '{{ program.contactEmail }}', '{{ program.contactName }}', '{{ program.defaultLocation }}', '{{ program.id }}', '{{ program.instagramUrl }}','{{ program.facebookUrl }}', '{{ program.bereaUrl }}')">Update Program Information</button>
+                    <script 
+                      type="module"> import { populateModal } from "/static/js/userManagement.js";
+                      window.programModal = function(programName, programDescription, partner, contactEmail, contactName, location, programId, instagramUrl, facebookUrl, bereaUrl){
+                      populateModal(programName, programDescription, partner, contactEmail, contactName, location, programId, instagramUrl, facebookUrl, bereaUrl)
+                      }
+                      //This script imports the populateModal function set the input boxes in the modals to program information
+                      //The program info is gotten from the programs list in userManagement.py and is passed using jinja on button click
+                    </script>
+                 </td>
               </tr>
-            </form>
-            {% endfor %}
-        </table>
+              {% endfor %}
+           </table>
+        </div>
+     </div>
+  </div>
+</div>
+</div>
+<div>
+<!--modal begins here-->
+<div class="modal fade" id="adminProgramManagement">
+  <div class="modal-dialog modal-dialog-scrollable" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Update Program Information</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
       </div>
+      <form id="updateProgramForm" class="info" action="" method="POST" enctype="multipart/form-data">
+        <div class="modal-body">
+            <tr>
+              <!-- Left modal column -->
+              <td class="cellSpacing">
```

Comment:
`<td>` is something that tables use. You want divs (probably), with bootstrap rows and cols

Reply:
Changed it to divs